packages/mural-canvas/src/lib/rpc.ts

```diff
@@ -181,6 +193,19 @@ export default class RpcClient extends EventEmitter {
     this.dispatch();
   };
 
+  rpc_callback(message: RpcMessage, error?: string, context?: RpcContext) {
+    if (!this.config) throw new Error('RpcClient not initialized');
+
+    const callback: RpcCallback = {
+      type: 'mural.rpc_callback',
+      rpcid: message.rpcid,
+      error,
+      context,
```

Comment:
**quesiton**: I wonder if we should sent the current `context` as part of the `rpc_callback`.

Reply:
We could do that and perhaps merge in the contents of the `context` parameter.

Alternatively, we could leave control to the client code--they can already explicitly pass the current `context` from `RpcClient`.